src/hooks.js

```diff
@@ -297,25 +300,18 @@ const onOpenPull = async context => {
 		return;
 	}
 
-	const { body, event } = formatReview( lintState, diffMapping );
-	console.log( {
-		owner,
-		repo,
-		number:    payload.number,
-		commit_id: commit,
-		body:      body,
-		event,
-	} );
+	const { body, event, comments } = formatReview( lintState, diffMapping );
 
 	// path, position, body
 
 	github.pullRequests.createReview( {
 		owner,
 		repo,
 		number:    payload.number,
-		commit_id: commit,
+		// commit_id: commit,
 		body:      body,
-		event,
+		 comments,
+		 event
```

Comment:
Invalid indentation here.

```suggestion
		comments,
		event
```